vignettes/join-keys.Rmd

```diff
@@ -150,6 +150,26 @@ jk["ds1", "ds5"] <- "a_column"
 jk["ds4", "ds5"] <- NULL
 ```
 
+## Merging Join Keys
+
+The consolidation of multiple _joining keys_ is achieved using the `c()` generic function, which generates a symmetric and valid `join_keys` object. When encountering identical relationship pairs, this operation retains the last occurrence of the pair within the specified argument order.
```

Comment:
I think "combining" would be a better word to "consolidation", the latter seems very business-y to me.

Reply:
Trying to avoid defining "merging" with "merging x and y" and lead to this. Great suggestion